docs/user/docs/index.md

```diff
@@ -7,64 +7,34 @@
 
 Welcome onboard of the Modeling Collaboration Manager. This platform helps you
 co-work on architectures using tools like
-[Capella](https://www.eclipse.org/capella/) and
-[Papyrus](https://www.eclipse.org/papyrus/).
+[Capella](https://www.eclipse.org/capella/),
+[Papyrus](https://www.eclipse.org/papyrus/),
+[pure::variants](https://www.pure-systems.com/purevariants) and
+[Jupyter](https://jupyter.org/) in combination with
+[py-capellambse](https://github.com/DSD-DBS/py-capellambse).
 
 This platform wraps the modeling tools to create an environment where users can
-work directly in the project context without having to install or configure the
-tool. It also takes care of housekeeping for the modeling projects themselves.
-For example, a automatic backup from
-[TeamForCapella](https://www.obeosoft.com/en/team-for-capella) to any Git
-repository can be set up.
-
-## Ways of co-working on a Modeling Project
-
-In the context of Capella the platform supports 2 main ways of co-working:
-
-<!-- prettier-ignore-start -->
-
-- **Git-only** - the modeling team uses a git repository to work on the model.
-  The team may use git branches to work on features or capabilities in parallel
-  and a main branch is used for integration and release-tagging. This way of
-  working gives the modeling leads / change control board great control over
-  what contents make it to the model that is used for releases (of things like
-  design documentation). On the downside this co-working method is fairly
-  complicated and requires skilled modeling leadership for challenges like
-  merge conflict resolution and fragmentation management (a way to break up
-  model into smaller files to reduce density of merge conflicts).
-- **Git + TeamForCapella** - with this approach the modeling team can co-work
-  with a very high degree of concurrency and stay away from the difficulties of
-  merge conflict resolution. On the downside it is much harder to control what
-  makes it into the model as there is no barrier except for maybe a modeling
-  process that would stop a person from making changes that are not allowed.
-  Yet there are a few ways around that limitation. For teams with basic or no
-  experience in modeling and git this is probably the best way to start
-  co-woking. Git is still used for nightly backup of the model and
-  release-tagging.
-
-    !!! info "Info: TeamForCapella license required"
-        For this co-working method to be enabled you need a valid
-        TeamForCapella license and TeamForCapella server installed and
-        integrated with Collab-Manager.
-
-<!-- prettier-ignore-end -->
+can work directly in the project context without having to install or configure
+the tool. It also takes care of the housekeeping of the modeling projects
+themselves. And that's not all: We also offer project access management and the
+site is completely responsive - and thus also usable on smartphones.
```

Comment:
```suggestion
themselves. And that's not all: We also offer project access management and since the site is completely responsive it is also usable on smartphones.
```